likes/views.py

```diff
@@ -1,3 +1,54 @@
-from django.shortcuts import render
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import F
 
-# Create your views here.
+from products.models  import Product
+from members.utils    import login_decorator
+from .models          import Like
+
+class LikesView(View):
+    @login_decorator
+    def get(self,request):
+        items = [{
+            "name"      : like.product.name,
+            "price"     : like.product.price,
+            "thumbnail" : like.product.thumbnail
+        } for like in Like.objects.filter(member_id=request.member)]
+
+        return JsonResponse({"ITEMS": items}, status=200)
+
+    @login_decorator
+    def post(self, request, product_id):
+        if not product_id.isdigit():
+            return JsonResponse({"ERROR": "NEED_NUMBER"}, status=400)
+
+        if not Product.objects.filter(id=product_id).exists():
+            return JsonResponse({"ERROR": "PRODUCT_NOT_EXIST"}, status=400)
+
+        if Like.objects.filter(member_id=request.member, product_id=product_id).exists():
+            return JsonResponse({"ERROR": "ALREADY_LIKED"}, status=400)
+
+        Like.objects.create(member_id=request.member.id, product_id=product_id)
+
+        product = Product.objects.get(id=product_id)
+        product.like_number = F("like_number") + 1
```

Comment:
```suggestion
        product.like_number +=  1
```
혹시 이렇게는 안 되나요?? 
F를 안 써도 될 것 같아서요

Reply:
F객체 없애서 다시 푸쉬했습니다!!